src/Uno.UI.RuntimeTests/Tests/HotReload/Frame/HRApp/Tests/Given_Frame.cs

```diff
@@ -29,7 +29,7 @@ public class Given_Frame : BaseTestClass
 	[TestMethod]
 	public async Task Check_NoChange_Page1()
 	{
-		var ct = new CancellationTokenSource(TimeSpan.FromSeconds(10)).Token;
+		var ct = new CancellationTokenSource(TimeSpan.FromSeconds(30)).Token;
```

Comment:
Do we still need those changes?

Reply:
I don't know. Will revert and see if CI passes

Reply:
It was needed. It may be some specific test that needs the more timeout, but it will take time to figure out which test specifically. It shouldn't be a big of problem anyways.